packages/core/core/src/TargetResolver.js

```diff
@@ -74,7 +80,7 @@ export default class TargetResolver {
         targets = [
           {
             name: 'default',
-            distDir: 'dist',
+            distDir: path.resolve(DEFAULT_DIST_DIRNAME),
```

Comment:
hmm, should this be relative to the package.json file instead?

@padmaia @devongovett

Reply:
Probably. `cwd` doesn't feel quite right.

Reply:
For `serve`, what do you think about using a temp directory for dist? I'd rather not write to the user's project unexpectedly, especially if they weren't expecting a directory named `dist`.

cc @devongovett

Reply:
It's nice to be able to easily inspect the files that parcel generates. Also, you'd get a `dist` directory when building for prod, so you'd need it in your `.gitignore` anyway so probably not a big deal if it shows up in dev as well.

Reply:
and yeah... I think it should be relative to package.json if it was declared there.

Reply:
It's possible the user has configured a different output directory than `dist` for their other targets. This is for one we come up with on the fly for serve. Should we just use whatever they specify for another target? 😬